web/static/styles/app.css

```
@@ -2699,6 +2763,22 @@ body.page-settings .bb-toggle-slider:before {
   align-items: end;
 }
 
+@media (max-width: 900px) {
```

Stylelint (17.2.0)

[error] Expected "context" media feature range notation (media-feature-range-notation)

(media-feature-range-notation)